### PR TITLE
VP-2064: Preview page button open wrong URL

### PR DESCRIPTION
**Pull request by pushnitsa**
### Problem
On content page edit blade preview button, redirects on wrong store url **when using Azure blob storage provider**.

### Solution
Preview button opened store url list blade, which using browser local storage for storing store url list, and after lick on one of the url, user should be redirected with correct url.

### Proposed of changes
Implemented store url list blade and fix relative urls.

### Additional context (optional)
store_url_list

### Make sure these boxes are checked:
- [x] Check all the changes in github PR - files count (non of them are redundant, have meaningful changes, all are added), if target branch is correct
- [x] Check methods and variable namings - it should be self descriptive, no typos
- [x] Check you did not introduce breaking changes in API and public models/services.
- [x] Respect extensibility - https://community.virtocommerce.com/t/extensibility-basics-the-domain-model-and-persistence-layer-extension/141
- [x] Follow [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and [SOLID](https://en.wikipedia.org/wiki/SOLID) principles
- [x] For unit tests - follow Microsoft best practices: https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices
- [x] Consolidate solution dependencies in case you are using newer version, update VC module dependencies in module.manifest. Do not upgrade 3rd party packages that are shipped with the platform with the version newer than in the platform.
- [x] Check code style conventions - https://github.com/VirtoCommerce/styleguide/blob/master/csharp.md
- [x] Check PR have a concise and descriptive title, follow [git commit message rules](https://github.com/VirtoCommerce/styleguide/blob/master/gitcommits.md)

### CODE REVIEW

File: src/VirtoCommerce.ContentModule.Azure/AzureContentBlobStorageProvider.cs [hunk 1]
```diff
@@ -57,7 +57,15 @@ public override async Task RemoveAsync(string[] urls)
         public override async Task<BlobEntrySearchResult> SearchAsync(string folderUrl, string keyword)
         {
             folderUrl = NormalizeUrl(folderUrl);
-            return await base.SearchAsync(folderUrl, keyword);
+
+            var result = await base.SearchAsync(folderUrl, keyword);
```
Comment (Vectorfield4) [inline src/VirtoCommerce.ContentModule.Azure/AzureContentBlobStorageProvider.cs:61]: Can you rename `result` to something more informative?

File: src/VirtoCommerce.ContentModule.Azure/AzureContentBlobStorageProvider.cs [hunk 2]
```diff
@@ -57,7 +57,15 @@ public override async Task RemoveAsync(string[] urls)
...
+
+            foreach (var blobEntry in result.Results)
+            {
+                blobEntry.RelativeUrl = blobEntry.RelativeUrl.Replace($"/{_options.RootPath.Replace('\\', '/')}", string.Empty);
```
Comment (Vectorfield4) [inline src/VirtoCommerce.ContentModule.Azure/AzureContentBlobStorageProvider.cs:65]: Can you move `"/{_options.RootPath.Replace('\\', '/')}"` outside of loop?

**Review by Vectorfield4 — APPROVED** [review]

File: src/VirtoCommerce.ContentModule.Azure/AzureContentBlobStorageProvider.cs (hunk 1 above)
Comment (AliveMen) [reply src/VirtoCommerce.ContentModule.Azure/AzureContentBlobStorageProvider.cs:61]: I think that `result` suitable for this place. You can see returning type in method signature.

Comment (pushnitsa) [reply src/VirtoCommerce.ContentModule.Azure/AzureContentBlobStorageProvider.cs:61]: I think it's good as is

**Review by AliveMen — APPROVED** [review]

**Review by Vectorfield4 — APPROVED** [review]

**Pull request merged**